### PR TITLE
add lightweight http healthcheck

### PR DESCRIPTION
**Pull request by sholdee**
Adds a basic http health check endpoint
Leverages existing build env, no added dependencies
Only ~3.5 MB increased image size
Enables use of load balancer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,6 +134,14 @@ spec:
           - containerPort: 51820
             protocol: UDP
             name: wireguard
+          - containerPort: 8080
+            protocol: TCP
+            name: healthcheck
+          livenessProbe: &probe
```
Comment (masipcat) [inline README.md:140]: is this a typo?

**Review by masipcat — APPROVED** [review]
Looks good to me. Thank!

File: README.md (hunk 1 above)
Comment (sholdee) [reply README.md:140]: This is correct. The example uses a yaml anchor `&probe` and alias `*probe` to set live and ready probes with identical parameters.

Comment (masipcat) [reply README.md:140]: nice, I didn't know about yaml anchors

**Review by masipcat — APPROVED** [review]

**Pull request merged**